app/javascript/angular/code/services/google/_map.js

```diff
@@ -311,11 +311,15 @@ export const removeMarker = function(marker) {
   marker.setMap(null);
 };
 
-export const drawMarker = function({ position, title, zIndex, icon }) {
-  var newMarker = new google.maps.Marker({ position, title, zIndex, icon });
+export const drawCustomMarker = ({ position }) => {
```

Comment:
There already is a method `drawCustomMarker ` in this file, the one you edited on line 196. If that's on purpose then ignore my comment, just wanted to make sure that you noticed that